File: src/main/java/com/hiddewieringa/elevator/domain/elevator/command/CreateElevator.java
package com.hiddewieringa.elevator.domain.elevator.command;

import com.hiddewieringa.elevator.domain.elevator.model.ElevatorId;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

public class CreateElevator {

    @TargetAggregateIdentifier
    private final ElevatorId elevatorId;

    public CreateElevator(ElevatorId elevatorId) {
        this.elevatorId = elevatorId;
    }

    public ElevatorId getElevatorId() {
        return elevatorId;
    }
}

File: src/main/java/com/hiddewieringa/elevator/domain/elevator/command/CreateElevator.kt
package com.hiddewieringa.elevator.domain.elevator.command

import com.hiddewieringa.elevator.domain.elevator.model.ElevatorId
import org.axonframework.commandhandling.TargetAggregateIdentifier

class CreateElevator(@field:TargetAggregateIdentifier
                     val elevatorId: ElevatorId)
